blockbase/block-template-parts/header.html

```diff
@@ -1,5 +1,5 @@
 <!-- wp:group {"className":"site-header","layout":{"type":"flex"},"style":{"spacing":{"padding":{"bottom":"80px"}}}} -->
-<div class="wp-block-group site-header" style="padding-bottom:80px">
+<div class="wp-block-group site-header alignfull">
```

Comment:
The Block markup shows an 80px bottom padding but that was removed here.  It should either be removed in both places or kept in both places otherwise the block editor will complain.

We've been making a push elsewhere to keep the layout details in the templates (allowing the user to have more control of that) so I believe keeping the 80px here is more appropriate than moving it into the _header.scss file.  (I believe the padding-top is actually moved FROM the _header.scss to the template in another change I recently reviewed).

Reply:
I don't think adding alignfull only as a class is the correct markup. In fact I don't think flex group blocks allow for alignments but this is working on my local site. It's weird.

+1 on Jason's comments about the padding. I think you did it because it looks bad when we have a colored background, make it a margin instead, so we maintain the same spacing as we originally had

Reply:
`alignfull` is a GB class assigned to blocks that are aligned with the "Full width" option. The class works here because GB loads the related CSS but it's not the correct way of applying it.